administrator/components/com_joomgallery/models/category.php

```diff
@@ -654,6 +679,15 @@ protected function batchAccess($value, $pks)
         )
       {
         $table->load($pk);
+
+        // Trigger Event onJoomBeforeSave (Returnvalue: true or false)
+        // $table contains still the old values
+        $plugins = $this->_mainframe->triggerEvent('onJoomBeforeSave', array(_JOOM_OPTION.'.category', $table, false, array('access'=>$value)));
+        if(in_array(false, $plugins, true))
+        {
+          return false;
+        }
+
```

Comment:
A question. Isn't it neccessary to additionally trigger 'onContentBeforeSave' for the JoomFinderPlugin here?